File: C#/C# Advanced/Methods/05_LargerThanNeighbours/LargerThanNeighbours.cs
using System;
using System.Linq;

class LargerThanNeighbours
{
    static void Main()
    {
        int arrSize = int.Parse(Console.ReadLine());
        int[] numbers = new int[arrSize];

        numbers = Console.ReadLine().Split(new char[] { ' ' },
            StringSplitOptions.RemoveEmptyEntries).Select(int.Parse).ToArray();

        Console.WriteLine(CountLargerThanNeighbours(numbers));
    }

    static int CountLargerThanNeighbours(int[] arr)
    {
        int count = 0;

        for (int i = 1; i < arr.Length - 1; i++)
        {
            if (arr[i] > arr[i + 1] && arr[i] > arr[i - 1])
            {
                count++;
            }
        }

        return count;
    }
}

File: Javascript/Javascript Fundamentals/Functions/larger-than-neighbours.js
function solve(args) {
    var n = +args[0],
    arr = args[1].split(' '),
    k,
    count = 0;

    for(k = 0; k < n - 1; k += 1) {
        if(LargerThanNeighbours(k, arr)){
            count += 1;
        }
    }

    console.log(count);
    function LargerThanNeighbours(i, array) {
        if(+i === 0 || +i >= array.lenght - 1){
            return false;
        }

        if(+array[i] > +array[i - 1] && +array[i] > +array[i + 1]){
            return true;
        } else{
            return false;
        }
    }
}